impl/src/fmt.rs

```diff
@@ -55,7 +55,9 @@ impl Display<'_> {
                 }
                 'a'..='z' | 'A'..='Z' | '_' => {
                     let ident = take_ident(&mut read);
-                    Member::Named(Ident::new(&ident, span))
+                    let mut ident = parse_str::<Ident>(&ident).unwrap();
```

Comment:
Although this line does use `unwrap()`, it should only panic in the same cases as `Ident::new()` already does, i.e. invalid identifiers.